### PR TITLE
Document the behavior of IM rules and multi-value indicator documents

### PR DESCRIPTION
**Pull request by nastasha-solomon**
Contributes to https://github.com/elastic/security-docs/issues/3354.

[Preview](https://security-docs_4326.docs-preview.app.elstc.co/guide/en/security/master/rules-ui-create.html#create-indicator-rule) - inserted a note after step 2e

### CODE REVIEW

File: docs/detections/rules-ui-create.asciidoc [hunk 1]
```diff
@@ -195,6 +196,9 @@ the indicator index patterns. The default query `@timestamp > "now-30d/d"` searc
 .. *Indicator mapping*: Compares the values of the specified event and indicator field
 values. When the field values are identical, an alert is generated. To define
 which field values are compared from the indices add the following:
++
+NOTE: Only single-value fields are supported.
```
Comment (rylnd) [inline docs/detections/rules-ui-create.asciidoc:200]: 👏

**Review by rylnd — DISMISSED** [review]
Changes are straightforward and LGTM!

File: docs/detections/rules-ui-create.asciidoc (hunk 1 above)
[the remark this answers is not in the document]
Comment (nastasha-solomon) [reply docs/detections/rules-ui-create.asciidoc:200]: If I end up moving the note up, I'm might make some minor revisions to the **Indicator mapping** description so it doesn't appear to conflict with the note. Maybe something like this:

> Compares the values of the specified event and indicator fields, and generates an alert if the values are identical. 

@rylnd is the revised version still correct?

[the remark this answers is not in the document]
Comment (rylnd) [reply docs/detections/rules-ui-create.asciidoc:200]: Yep, that works for me! 👍

[the remark this answers is not in the document]
Comment (nastasha-solomon) [reply docs/detections/rules-ui-create.asciidoc:198]: Resolved in [10dc3d9](https://github.com/elastic/security-docs/pull/4326/commits/10dc3d9c0372e38a72138c27a61e29816f702573).

**Review by natasha-moore-elastic — APPROVED** [review]
Looks good, thanks!

**Pull request merged**